src/Compactable.ts

```diff
@@ -200,6 +181,26 @@ export function separate<F, G>(
   return (fge) => separated(_compact(pipe(fge, _map(getLeft))), _compact(pipe(fge, _map(getRight))))
 }
 
+// -------------------------------------------------------------------------------------
+// backwards compatibility
+// -------------------------------------------------------------------------------------
+
+export {
+  /**
+   * Re-exported interface for backwards compatibility.
+   *
+   * @since 2.10.0
+   */
+  Separated,
+
+  /**
+   * Re-exported constructor for backwards compatibility
+   *
+   * @since 2.10.0
+   */
+  separated
```

Comment:
this can be removed as was added for 2.10.0

Reply:
I believe this will break quite a few modules, as they are importing `separated` from `Compactable` (Either, Filterable, ReadonlyArray/Map/Set/Record)

Reply:
They should import `separated` from the new `Separated` module